AppSettingsManagerBff/AppSettingsManagerBff.Infrastructure/ApiRepositories/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace AppSettingsManagerBff.Infrastructure.ApiRepositories;
+
+public static class Constants
+{
+    private const string RequestMediaType = "application/json";
+
+    private static readonly JsonSerializerOptions JsonSerializerOptions = new JsonSerializerOptions
+    {
+        PropertyNameCaseInsensitive = true
+    };
+
+    public static StringContent GetStringContent(object request) =>
+        new StringContent(JsonSerializer.Serialize(request), Encoding.UTF8, RequestMediaType);
+
+    public static async Task<T> DeserializeResponse<T>(HttpResponseMessage response) =>
```

Comment:
Nice! always good to pull any code that is repeated more than once into its own method